scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/descriptor/JobDescriptorImpl.java

```diff
@@ -169,45 +188,19 @@ private void makeTree(InternalJob job) {
      * a startable task : has no dependency, and is not target of an if control flow action
      *
      * @param t a Task
-     * @param otherTasks the other tasks contained in the job containing task t
+     * @param taskNames set of task names which cannot be entry point
      * @return true if t is an entry point among all tasks in otherTasks, or false
      */
-    private boolean isEntryPoint(InternalTask t, List<InternalTask> otherTasks) {
+    private boolean isEntryPoint(InternalTask t, Set<String> taskNames) {
         List<TaskState> deps = t.getDependences();
         boolean entryPoint = false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "entryPoint" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)